sympy/calculus/euler.py

```diff
@@ -3,23 +3,24 @@
 Euler-Lagrange Equations for given Lagrangian.
 """
 from itertools import combinations_with_replacement
+
 from sympy import Function, sympify, diff, Eq, S, Symbol, Derivative
-from sympy.core.compatibility import (iterable, range)
+from sympy.core.compatibility import iterable, range
```

Comment:
Yes, refactoring the codebase seemed like something easy to do, but it isn't that trivial. In other places there are other codestyle issues that are not so easy to fix - `function too complex` warning is among them and it requires some more work to fix it.